src/microbe_stage/editor/ToleranceOptimalMarker.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Godot;
+
+/// <summary>
+///   Displays additional tolerance slider info, such as the optimal value and value labels.
+/// </summary>
+public partial class ToleranceOptimalMarker : Control
+{
+#pragma warning disable CA2213
+    [Export]
+    private Texture2D? markerTextureOverride;
+
+    [Export]
+    private TextureRect optimalValueMarker = null!;
+#pragma warning restore CA2213
+
+    private bool showMarker = true;
+
+    private float optimalValue;
+
+    public bool ShowMarker
+    {
+        get => showMarker;
+        set
+        {
+            if (value == showMarker)
+                return;
+
+            showMarker = value;
+
+            optimalValueMarker.Visible = value;
+        }
+    }
+
+    public float OptimalValue
```

Comment:
So this seems like the horizontal position in pixel units? I think the name should be changed as this doesn't seem to relate to the slider values?